.travis.yml

```diff
@@ -73,7 +73,7 @@ script:
   - |
     if [[ -v PRODUCT_TESTS_BASIC_ENVIRONMENT ]]; then
       presto-product-tests/bin/run_on_docker.sh \
-        multinode -x quarantine,big_query,storage_formats,profile_specific_tests,tpcds
+        multinode -x quarantine,big_query,storage_formats,profile_specific_tests,tpcds,cassandra,mysql_connector,postgresql_connector,mysql
```

Comment:
why do we have `mysql` and `mysql_connector`? It would be nice to merge them. Maybe something for following PR.

Reply:
Yeah, i noticed that. We should definitely do that as part of the follow up.

Reply:
please file a ticket for that, to not to forget about it

Reply:
https://github.com/prestodb/presto/issues/10439#issuecomment-382531616